cli-plugins/apps/getting-started/configuration.hbs.md

```diff
@@ -0,0 +1,168 @@
+# Configuration
+
+## <a id='changing-clusters'></a>Changing clusters with --context
+
+The Apps CLI plug-in refers to the default kubeconfig file to access a Kubernetes cluster.
+When you run a `tanzu apps` command, the plug-in uses the default context that is defined in that
+kubeconfig file (located by default at `HOME/.kube/config`).
+
+There are two ways to change the target cluster:
+
+1. Use `kubectl config use-context CONTEXT-NAME` to change the default context. All subsequent
+`tanzu apps` commands target the cluster defined in the new default kubeconfig context.
+
+2. Include the `--context CONTEXT-NAME` flag when running any `tanzu apps` command.
+
+   >**Note** Any subsequent `tanzu apps` commands that do not include the `--context CONTENT-NAME`
+     flag continue to use the default context set in the kubeconfig.
+
+## <a id='override-kubeconfig'></a>Overriding the default kubeconfig
+
+There are two approaches to overriding the default kubeconfig:
+
+1. Set the environment variable `KUBECONFIG=PATH` to change the kubeconfig the Apps CLI plug-in will
+   reference.
+
+   All subsequent `tanzu apps` commands reference the non-default kubeconfig assigned to the environment
```

Comment:
to be consistent with how the same sentence was included for a similar situation (see line 11 above), start this sentence without a double-new-line (start the sentence immediately following "Set the environment variable `KUBECONFIG=PATH` to change the kubeconfig the Apps CLI plug-in will
   reference." (without a new line - just like the pattern in line 11).

Reply:
changed